.circleci/config.yml

```diff
@@ -223,6 +224,9 @@ jobs:
                   command: |
                       cd demos/showcase
                       yarn prettier:check
+            - run:
+                  name: 'Link Components'
+                  command: yarn link:components
```

Comment:
Is this needed? Installing the local version of components is already handled on line 216.

Reply:
it was simply saying the files are not found and failing the pipeline

Reply:
Can you confirm what files from circleCI build job. I only have seen snapshot tests failing

Reply:
I'm curious about this too...the build folder should be there.

We build the components in the build_library job and then pack the dist folder into a tgz, which we then preserve in the workspace for the subsequent job:
![image](https://github.com/etn-ccis/blui-react-native-component-library/assets/29152776/6773efa1-6fba-44f8-b4e4-3d70c900cbf5)

And then we install the components into the showcase from that packaged version:
![image](https://github.com/etn-ccis/blui-react-native-component-library/assets/29152776/66599f5f-de4d-442f-be0c-30a8e7c49669)

Unless the relative path got messed up, the files should be there.

Reply:
the component Package.json did not include md3 folder so the npm-pack was leaving md3 folder out of this and creating those errors
Fixed that